cmd/ktor/main.go

```diff
@@ -109,6 +100,33 @@ func main() {
 		}
 
 		command.Generate(client, result.ProjectDir, result.ProjectName, result.Plugins, verboseLogger, hasGlobalLog, ctx)
+	case cli.OpenAPI:
```

Comment:
It would be great to extract a named methods for this cases, so we can look them up by name

Reply:
We have them in the `command` package. Also, it's fairly easy to find a command by using the corresponding constantans which are defined in one place.